src/app/src/components/FishNames.js

```diff
@@ -11,20 +11,23 @@ const StyledFishNames = styled(Flex)`
 `;
 
 const CommonName = styled(Heading)`
+    font-family: ${themeGet('font')};
     margin-bottom: 0;
     line-height: ${themeGet('lineHeights.compact')};
-    font-size: 2rem;
 `;
 
 const ScientificName = styled(Text)`
-    font-style: italic;
+    font-family: ${themeGet('font')};
+    font-style: ${themeGet('fontStyles.italic')};
     margin-bottom: ${themeGet('space.medium')};
 `;
 
-const FishNames = ({ commonName, scientificName }) => {
+const FishNames = ({ className, commonName, scientificName }) => {
```

Comment:
I didn't see a use case of this component where we pass in className at the same time. Is this a vestigial change?

Reply:
I looked into this and can't really figure out why it's working, but it is. For the sake of time, and because this code is not foreseeably dangerous, we're leaving it.